example/.gitignore

```diff
@@ -20,7 +20,6 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
-project.xcworkspace
```

Comment:
why remove this?

Reply:
`yarn-lock` should be added

Reply:
With `yarn-lock` you mean `yarn.lock`? `yarn.lock` records a _known_ dependency tree for this example project (exact dependency versions that the project is guaranteed to work with). Please check this article for more info:
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/